packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -198,14 +209,17 @@ export class ColumnHeaderCell extends AbstractComponent<IColumnHeaderCellProps,
             pin: true,
             to: "window",
         }];
+        const classes = classNames(Classes.TABLE_TH_MENU_CONTAINER, {
+            [Classes.TABLE_TH_MENU_OPEN]: this.state.isActive,
```

Comment:
when does `this.state.isActive` change? i don't see any `setState`s in this PR...